projects/aws/eks-anywhere-packages/Makefile

```diff
@@ -40,6 +40,8 @@ HELM_CHART_NAMES=eks-anywhere-packages-migrations eks-anywhere-packages-crds eks
 
 BUILDSPEC_COMPUTE_TYPE=BUILD_GENERAL1_LARGE
 
+EXCLUDE_FROM_UPGRADE_BUILDSPEC=true
```

Comment:
Id probably keep this along with all the packages since what would it hurt? But we can review all these skipped ones as we go and use this automation for others

Reply:
Yeah the idea is to revisit these once the job stabilizes.